api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -306,7 +285,7 @@
 
     // The Welder extension offers direct links to/from playground mode; write the AoU config file
     // to both locations so notebooks will work in either directory.
-    String aouConfigUri = aouConfigDataUri(fcWorkspace, cdrVersion, projectName);
+    String aouConfigUri = aouConfigDataUri(firecloudWorkspace, cdrVersion, workspaceNamespace);
```

Comment:
Above @ line 284, I think we can have a better name than `localizeMap`, e.g. `bucketPathToNotebookPath` if I'm reading things right.